src/ledger.test.js

```diff
@@ -0,0 +1,62 @@
+import describe from 'riteway';
+
+import { ethereum as $ } from './index';
+import { $$, addPercent, subtractPercent } from './ledger';
+
+describe('$$', async assert => {
+  const should = 'total $(x) amounts';
+  /* eslint-disable */
+  const actual = $$(
+    $('40'),
```

Comment:
Why string is there a reason we use strings (`'40'`) instead of numbers (`40`) for these tests? 🧐

Reply:
Yes. The JavaScript number type is limited to 16 decimal digits of precision. We need to support at least 18 for ethereum support. Other cryptocurrencies can have arbitrary precision. The best way to get that precision in JavaScript using a literal format is to use string literals. We should write dedicated tests for that, but I'm trying to get this out quickly so I can start using it.